src/defs/deprecated.ts

```diff
@@ -37,1035 +37,1035 @@ DeprecatedDefinitions.push(tmpDef);
+tmpDef.returnType = "bool";
+tmpDef.args = ["player thePlayer", "int weapon", "int ammo"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 DeprecatedDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
 tmpDef.label = "toggleCameraFixedMode";
-tmpDef.description = deprecatedText + "Please use ???";
-tmpDef.returnType = "";
-tmpDef.args = [];
+tmpDef.description = deprecatedText + "Please use setCameraPosition and setCameraLookAt functions respectively";
+tmpDef.returnType = "bool";
+tmpDef.args = ["bool fixed"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 DeprecatedDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
 tmpDef.label = "unbanIP";
-tmpDef.description = deprecatedText + "Please use ???";
-tmpDef.returnType = "";
-tmpDef.args = [];
+tmpDef.description = deprecatedText + "Please use removeBan";
+tmpDef.returnType = "bool";
+tmpDef.args = ["string ipToUnban", "[player unbanningPlayer = nil]"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 DeprecatedDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
 tmpDef.label = "unbanSerial";
-tmpDef.description = deprecatedText + "Please use ???";
-tmpDef.returnType = "";
-tmpDef.args = [];
+tmpDef.description = deprecatedText + "Please use removeBan";
+tmpDef.returnType = "bool";
+tmpDef.args = ["string serialToUnban"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 DeprecatedDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
 tmpDef.label = "warpPlayerIntoVehicle";
-tmpDef.description = deprecatedText + "Please use ???";
-tmpDef.returnType = "";
-tmpDef.args = [];
+tmpDef.description = deprecatedText + "Please use warpPedIntoVehicle";
+tmpDef.returnType = "bool";
+tmpDef.args = ["player thePlayer", "vehicle theVehicle", "[ int seat=0 ]"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 DeprecatedDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
 tmpDef.label = "xmlCreateSubNode";
-tmpDef.description = deprecatedText + "Please use ???";
-tmpDef.returnType = "";
-tmpDef.args = [];
+tmpDef.description = deprecatedText + "Please use xmlCreateChild";
+tmpDef.returnType = "xmlnode";
+tmpDef.args = ["xmlnode parentNode", "string tagname"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 DeprecatedDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
 tmpDef.label = "xmlFindSubNode";
-tmpDef.description = deprecatedText + "Please use ???";
-tmpDef.returnType = "";
-tmpDef.args = [];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Shared;
-DeprecatedDefinitions.push(tmpDef);
\ No newline at end of file
+tmpDef.description = deprecatedText + "Please use xmlFindChild";
+tmpDef.returnType = "xmlnode";
+tmpDef.args = ["xmlnode parent", "string subnode", "int index"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Shared;
+DeprecatedDefinitions.push(tmpDef);
+
+// tmpDef = new MTAFunction;
+// tmpDef.label = "onClientChangeNick";
```

Comment:
Why was this removed? https://wiki.multitheftauto.com/wiki/OnClientChangeNick

Reply:
Well there isn't any support for events so I just commented it for the time being since it would give the wrong format.